api/src/main/java/openconsensus/trace/export/SpanData.java

```diff
@@ -0,0 +1,396 @@
+/*
+ * Copyright 2017, OpenConsensus Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package openconsensus.trace.export;
```

Comment:
I would move this into data package.

Reply:
See other comment. Scenario we want to enable is for instrumentation to use API package to record `SpanData`.

Reply:
I understand the reasoning, just suggested to have SpanData in this directory: https://github.com/bogdandrutu/openconsensus/tree/master/api/src/main/java/openconsensus/trace/data

Reply:
makes sense than.